libmscore/rest.cpp

```diff
@@ -924,30 +942,57 @@ void Rest::localSpatiumChanged(qreal oldValue, qreal newValue)
       }
 
 //---------------------------------------------------------
-//   getProperty
+//   propertyDefault
 //---------------------------------------------------------
 
-QVariant Rest::getProperty(Pid propertyId) const
+QVariant Rest::propertyDefault(Pid propertyId) const
       {
       switch (propertyId) {
             case Pid::GAP:
-                  return _gap;
+                  return false;
+            case Pid::YPOS_MMREST_NUMBER:
+                  return score()->styleV(Sid::yPosMMRestNumber);
             default:
-                  return ChordRest::getProperty(propertyId);
+                  return ChordRest::propertyDefault(propertyId);
             }
       }
 
+//————————————————————————————
+//   resetProperty
+//————————————————————————————
+
+void Rest::resetProperty(Pid id)
+      {
+      setProperty(id, propertyDefault(id));
```

Comment:
is that necessary? Isn't there already a `resetProperty` among the parents? Just asking, haven't looked it up.

Reply:
There is not that I could find.